Playwright.Axe/AxeFrameContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿#nullable enable
+
+namespace Playwright.Axe
+{
+    /// <summary>
+    /// Run Context for a Frame
+    /// </summary>
+    public sealed class AxeFrameContext
+    {
+        /// <summary>
+        /// Selector for the Frame in question.
+        /// </summary>
+        public AxeCrossTreeSelector FrameSelector { get; }
+
+        /// <summary>
+        /// Run Context for the Frame.
+        /// </summary>
+        public AxeSerialContext FrameContext { get; }
```

Comment:
I don't think the `AxeSerialContext` typing here is complete enough to be usable. The `frameContext` returned by `getFrameContexts` is *not* exactly the same type as the `context` argument passed by a user to `axe.run` - it includes several implementation details that are required for the correct operation of axe in child frames.

Axe doesn't document exactly what properties the frame context contains, only that it should be passed as-is to `runPartial`. I think the safer option here would be to intentionally avoid deserializing the `frameContext` property to C# object types and instead just keep it as opaque JSON that can be passed as-is to the `runPartial` calls in child frames.